docs/user/kubevirt/install_ipi.md

```diff
@@ -0,0 +1,156 @@
+![KubeVirt Logo](./images/kubevirt-image.jpg)
+
+**Table of Contents**
+
+- [Install using KubeVirt platform provider](#install-using-kubevirt-platform-provider)
+  * [Overview](#overview)
+  * [Prerequisite](#prerequisite)
+  * [Minimum resources](#minimum-resources)
+  * [Install](#install)
+    + [Minimum permission for installation](#minimum-permission-for-installation)
+    + [kubevirt-credentials](#kubevirt-credentials)
+    + [Bootstrap VM](#bootstrap-vm)
+    + [Install using the wizard](#install-using-the-wizard)
+    + [Install using Install Config yaml](#install-using-install-config-yaml)
+    + [Run the installer](#run-the-installer)
+    + [After installation is done](#after-installation-is-done)
+
+# Install using KubeVirt platform provider 
+
+## Overview
+
+This provider enables the OpenShift Installer to provision VM resources on
+Kubernetes platform which includes KubeVirt virtualization, that will be used as worker and masters of the clusters.
+As part of the installation process, it will also create the bootstrap machine.
+
+## Prerequisite
+
+1. Installed Infra cluster, using [Kubernetes](https://kubernetes.io/) or [OpenShift](https://www.openshift.com/)<br/>
+   Prerequisite for Kubernetes:
+   - [KubeVirt](https://kubevirt.io/) installed in the cluster
+   - [Multus-cni](https://01.org/kubernetes/building-blocks/multus-cni) plugin installed in the cluster<br/>
+
+   prerequisite for OpenShift:
+   - [OpenShift Virtualization](https://www.openshift.com/learn/topics/virtualization/)
+
+2. Create the network that would be used by the tenant cluster:
+   - Create a bridge on the infra cluster nodes
+   - Create multus network (Network Attachment Definition) on the bridge
```

Comment:
Please add example yamls for the nad and the bridge

Reply:
As discussed, we will avoid adding those examples